src/main/java/org/folio/fqm/resource/FqlQueryController.java

```diff
@@ -34,6 +33,17 @@ public ResponseEntity<QueryDetails> getQuery(UUID queryId, Boolean includeResult
     return queryDetails.map(ResponseEntity::ok).orElseGet(() -> ResponseEntity.notFound().build());
   }
 
+  @Override
+  public ResponseEntity<List<Map<String, Object>>> getContents(UUID entityTypeId, List<String> fields, List<UUID> ids) {
```

Comment:
Right now this is a GET request with entityTypeId, fields, and ids as request parameters. Passing the list of fields and ids as request parameters is kind of messy, so we may want to consider doing this as a POST with a request body or something similar instead.

Reply:
I have a feeling you're right. If the list of fields or IDs is particularly large (which I'm sure it will be at some point), we'll run into the size limit on the URL in the GET. I'm kind of inclined to hold off until we know it's really a problem before we add a new endpoint, though

Fun fact: last I checked, there's no formally defined size limit for that in the HTTP RFCs, so it can (and does) change from one platform to the next 🙂